### DISCUSSION TITLE
Disabling Adaptive Lighting after a certain time

### DISCUSSION THREAD

**Discussion by b101marc** [basnijholt/adaptive-lighting#342] [Q&A]
Hello all,

I use Adaptive Lighting primarily during the day and in the evening I have a set scene that I prefer. Currently I have a binary sensor titled 'Daytime' set as a condition in a 'Adaptive Lighting Schedule' Automation which triggers my 'individual rooms to apply adaptive lighting hourly, until 'Daytime' turns off. I've had to do this because I've noticed sometimes certain lights just won't adapt randomly through the day, but it doesn't bother me as it gives me a greater degree of control.

I also have a Boolean toggle to apply adaptive in my dashboard should it need to be triggered manually for whatever reason, like if I don't want my usual scene applied if guests are over, need more light, etc...

This works great typically, however if any light turns on from an off state, those lights start to adapt despite it being out of schedule, which make sense logically. This is true if a motion sensor triggers a light on which triggers all my lights to start adapting again, despite each room being its own integration.

Long story short, is there a way to tell Adaptive Lighting to stop adapting after a certain hour? I'm likely overthinking this, but it's been driving me insane. If enabling/disabling integrations were possible that would be my easy fix, but alas.

Can provide whatever additional information is needed. Automation screenshot below.![image](https://user-images.githubusercontent.com/41177825/191893699-bb59ced0-249f-47ae-86c2-2f418373e9c1.jpeg)

Edit: Example of one of the room AL integrations, the one that gives me the most issues. All others are basically identical.![image](https://user-images.githubusercontent.com/41177825/191894805-b14d3e09-bf70-4b35-974f-d592101f59dd.jpeg)
![image](https://user-images.githubusercontent.com/41177825/191894823-f9ae0f6c-a294-4cb6-a5f5-fd4d02ff1e52.jpeg)

**Comment by th3w1zard1** [comment]
The way adaptive-lighting works is while it's enabled, it adapts your lights. Period.

Config options such as `take_over_control` and `detect_non_ha_changes` will attempt to search for manual events, but they require two light.turn_on calls to execute this.

If you want your scenes to fire manually controlled events while the integration is enabled, you need to setup your scene so it fires two light.turn_on calls: once to turn on the lights, and the next to set the correct brightness/color.

Your other option is turning off the main adaptive-lighting switch while your scene is active, then setup an automation that checks when the light is turned off, when it detects it Off your automation should turn the adapt switch back on (lights aren't adapted while they're off).
